go/test/endtoend/backup/mysqlctld/backup_mysqlctld_test.go

```diff
@@ -24,5 +24,5 @@ import (
 
 // TestBackupMysqlctld - tests the backup using mysqlctld.
 func TestBackupMysqlctld(t *testing.T) {
-	backup.TestBackup(t, backup.Mysqlctld, "", 0)
+	backup.TestBackup(t, backup.Mysqlctld, "xbstream", 0)
```

Comment:
I am not sure about this change either.

Reply:
this is because xtrabackup 8.0 with mysql8.0 does not support "tar" (which is default) but xbstream. This change is cherrypicked from https://github.com/vitessio/vitess/pull/10691/files#diff-909528cb082669b2422729be9acff461e293f142b88072415a517977d87ec76e